src/test/java/com/rybka/constant/ParameterValue.java

```diff
@@ -0,0 +1,13 @@
+package com.rybka.constant;
+
+public final class ParameterValue {
```

Comment:
Use this constants in the class level(per specific test cases) 
no need to create class. 

additionally ParameterValue bad naming, and USD, EUR, UAH as well.

Reply:
fixed